Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -1254,8 +1254,23 @@ def error_handler(self, res: Response):
             DemistoException: There is a problem with one or more arguments.
             DemistoException: One or more of the specified fields are invalid. Please validate them.
         """
-        output = res.json()
         error_code = res.status_code
+        if res.status_code == HTTPStatus.UNAUTHORIZED:
+            raise DemistoException(
+                "Authorization Error: make sure Username and Password are set correctly."
+            )
+        if res.status_code == HTTPStatus.NOT_FOUND:
+            raise DemistoException(
+                "Connection Error: Make sure server URL is set correctly."
+            )
+        output = {}
+        try:
+            output = res.json()
+        except requests.exceptions.JSONDecodeError:
+            raise DemistoException(
+                res,
```

Comment:
the first argument in the DemistoException c'tor is a message (i.e. should be a string)